cli/configuration/v1/flat_map.go

```diff
@@ -19,7 +19,7 @@ func FlatMap(configs ...*Config) *Config {
 		flattenCandidates := make([]*Config, 0)
 		for _, config := range config.Configurations {
 			var cfg Config
-			if err := scheme.Convert(config, &cfg); err != nil {
+			if err := Scheme.Convert(config, &cfg); err != nil {
```

Comment:
I wonder. Scheme has a map inside. This all will eventually end up on `setupOCMConfig` for example. Do you think scheme is thread-safe?

Reply:
I believe it is acceptable to consider that Scheme is not thread safe and thus FlatMap is also not threadsafe because the pattern to add things to a scheme is to do it once at startup